app/src/renderer/components/common/TmSession.vue

```diff
@@ -1,32 +1,34 @@
 <template>
-  <div class="tm-session-wrapper">
-    <session-loading v-if="session.modals.session.state == 'loading'" />
-    <session-welcome v-else-if="session.modals.session.state == 'welcome'" />
-    <session-sign-up v-else-if="session.modals.session.state == 'sign-up'" />
-    <session-sign-in v-else-if="session.modals.session.state == 'sign-in'" />
-    <session-account-delete
-      v-else-if="session.modals.session.state == 'delete'"
-    />
-    <session-hardware v-else-if="session.modals.session.state == 'hardware'" />
-    <session-import v-else-if="session.modals.session.state == 'import'" />
+  <modal v-if="active" :close="close">
+    <div slot="main">
+      <session-welcome v-if="session.modals.session.state == 'welcome'" />
+      <session-sign-up v-else-if="session.modals.session.state == 'sign-up'" />
+      <session-sign-in v-else-if="session.modals.session.state == 'sign-in'" />
+      <session-account-delete
+        v-else-if="session.modals.session.state == 'delete'"
+      />
+      <session-hardware v-else-if="session.modals.session.state == 'hardware'" />
```

Comment:
can we rename this to `session-ledger`? and have something like
```suggestion
      <session-ledger v-else-if="session.modals.session.state == 'ledger'" />
```

Reply:
yeah that would be more descriptive indeed